src/main/java/at/hannibal2/skyhanni/features/misc/chocolatefactory/ChocolateFactoryStats.kt

```diff
@@ -0,0 +1,80 @@
+package at.hannibal2.skyhanni.features.misc.chocolatefactory
+
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object ChocolateFactoryStats {
+
+    private val config get() = ChocolateFactoryApi.config
+
+    private var displayList = listOf<String>()
+
+    @SubscribeEvent
+    fun onBackgroundDraw(event: GuiRenderEvent.ChestGuiOverlayRenderEvent) {
+        if (!ChocolateFactoryApi.isEnabled()) return
```

Comment:
isEnabled is not needed here since the inChocolateFactory should cover it. (When suggestion is implemented in ChocolateFacoryApi at InventoryFullyOpenedEvent)
x5